{{cookiecutter.github_project_name}}/setup.py

```diff
@@ -39,22 +40,26 @@
 package_data_spec = {
     name: [
         'nbextension/static/*.*js*',
-        'labextension/*.tgz'
+        # 'labextension/*.tgz',
+        'labextension/*'
     ]
 }
 
+labext_name = "{{ cookiecutter.npm_package_name }}"
+
 data_files_spec = [
     ('share/jupyter/nbextensions/{{ cookiecutter.python_package_name}}',
         nb_path, '*.js*'),
-    ('share/jupyter/lab/extensions', lab_path, '*.tgz'),
+    ("share/jupyter/labextensions/%s" % labext_name, lab3_path, "*.*"),
```

Comment:
Similar to the JupyterLab extension cookiecutter:

```suggestion
    ("share/jupyter/labextensions/%s" % labext_name, lab3_path, "**"),
```